valhalla/jawn/src/lib/handlers/ResponseBodyHandler.ts

```diff
@@ -216,6 +224,34 @@ export class ResponseBodyHandler extends AbstractLogHandler {
     return responseBody;
   }
 
+  private isAssistantResponse(responseBody: any): boolean {
+    if (typeof responseBody !== "object" || responseBody === null) {
+      return false;
+    }
+    return (
+      responseBody.hasOwnProperty("assistant_id") ||
+      responseBody.hasOwnProperty("thread_id") ||
+      responseBody.data?.[0]?.hasOwnProperty("assistant_id") ||
+      responseBody.hasOwnProperty("metadata")
+    );
+  }
+
+  private determineAssistantModel(
+    responseBody: any,
+    currentModel?: string
+  ): { responseModel: string; model: string } {
+    if (
+      this.isAssistantResponse(responseBody) &&
+      responseBody.hasOwnProperty("status") &&
+      ["queued", "in_progress"].includes(responseBody.status)
+    ) {
+      return { responseModel: "Assistant Polling", model: "Assistant Polling" };
```

Comment:
nit - let's make these more "Contant Like" name and actually contants
```ts
const MODEL = "assistant-polling"; 
```
For example